src/gmail.d.ts

```diff
@@ -495,19 +501,19 @@ declare type GmailDomCompose = {
        options.type  string  to, cc, or bcc to check a specific one
        options.flat  boolean if true will just return an array of all recipients instead of splitting out into to, cc, and bcc
     */
-    recipients(options?: { type: string, flat: boolean }): GmailEmailAddress[];
+    recipients(options?: { type: 'to' | 'cc' | 'bcc', flat: boolean }): GmailDomComposeRecipients | string[];
     /**
       Retrieve the current 'to' recipients
      */
-    to(): string;
+    to(): string[];
     /**
       Retrieve the current 'cc' recipients
      */
-    cc(): string;
+    cc(): string[];
     /**
       Retrieve the current 'bcc' recipients
      */
-    bcc(): string;
+    bcc(): string[];
```

Comment:
These changes don't document what we're currently doing.but rather document what they *will* be with the changes to Gmail.js itself further down.

While so far not really documented, this *is* a public API, and I wouldn't want to change it needlessly. This could cause unexpected breakage for people upgrading.

Couldn't we just instead document what it currently does, i.e. document this as returning jQuery? To me, that sounds a lot safer.